tests/test_network.py

```diff
@@ -1,19 +1,70 @@
 import descarga_datos
 import os
 from descarga_datos import download_file_from_repo
+import pytest
 
 if not os.path.exists("./results"):
     os.mkdir("./results")
 
+test_data = [
+    ("tabular_data_packages", "captura_gatos_socorro.csv", True),
+    ("archivos_binarios", "clarion_vegetal_types.zip", True),
+    ("archivos_binarios", "XXclarion_vegetal_types.zipXX", False),
+]
 
-def test_download_file_from_repo():
-    archivo = descarga_datos.internals.DataFile(
+
+@pytest.mark.parametrize("repo, filename, is_a_file", test_data, ids=["CSV", "ZIP", "Not a file"])
```

Comment:
@devarops quedó a todo dar esta refactorizada:
![image](https://user-images.githubusercontent.com/35377740/161320884-64c8b88e-2685-4fc5-8c5b-f5cf450515ca.png)
Además los mensajes son descriptivos (CSV, ZIP y Not a file).